nuxeo-core/nuxeo-core-event/src/main/java/org/nuxeo/ecm/core/event/impl/EventListenerDescriptor.java

```diff
@@ -300,4 +300,10 @@ public boolean acceptBundle(EventBundle bundle) {
         return false;
     }
 
+    @Override
+    public String toString() {
+        return name;
+
```

Comment:
Empty line, also it could be better with more fields (use your ide toString generator).

Reply:
Yep, WDYT about:
```
EventListenerDescriptor [name=testPostCommit1, className=org.nuxeo.ecm.core.event.test.DummyPostCommitEventListener, isPostCommit=true, isAsync=false]
```